src/components/Messanger/Workspace/Message/AudioMessageContent/AudioMessageContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useRef, useState } from 'react';
+import { useWavesurfer } from '@wavesurfer/react';
+
+import styles from './audioMessageContent.module.css';
+import css from '../message.module.css';
+import { ReactComponent as PlayIcon } from '../../../../../assets/Messanger/RecordingAudio/PlayIcon.svg';
+import { ReactComponent as PauseIcon } from '../../../../../assets/Messanger/RecordingAudio/PauseIcon.svg';
+import { ReactComponent as PinnedIcon } from '../../../../../assets/Messanger/iconForPinnedMessage.svg';
+import { formatTime } from './lib';
+
+type Props = {
+  audioRef: React.RefObject<HTMLAudioElement>;
```

Comment:
```suggestion
  audioRef: RefObject<HTMLAudioElement>;
```
I never used RefObject before :)